kafka-rest/src/main/java/io/confluent/kafkarest/KafkaRestApplication.java

```diff
@@ -114,6 +116,13 @@ protected void setupInjectedResources(
     }
   }
 
+  @Override
+  protected void registerExceptionMappers(Configurable<?> config, KafkaRestConfig restConfig) {
```

Comment:
@norwood @rajinisivaram  KafkaExceptionMapper extends GenericExceptionMapper class.. To use KafkaExceptionMapper,  we need to override Application. registerExceptionMappers() method and register KafkaExceptionMapper. Otherwise GenericExceptionMapper will take priority.